docs/integrations/microsoft-azure/azure-functions.md

```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
+* Access detailed billing information and cost breakdowns.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Cost.png')} alt="Azure Functions Cost dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Error
+
+The **Azure Functions - Error** dashboard provides insights into errors encountered by your Azure Functions. This includes error rates, types of errors, and patterns over time.
+
+Use this dashboard to:
+* Track and investigate error occurrences in your Azure Functions.
+* Identify and diagnose recurring error patterns.
+* Review error logs and detailed reports to improve function reliability.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Error.png')} alt="Azure Functions Error dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Memory
+
+The **Azure Functions - Memory** dashboard provides information on memory usage and consumption by your Azure Functions. This includes metrics on memory allocation, usage patterns, and any potential memory issues.
+
+Use this dashboard to:
+* Monitor memory consumption and efficiency of your Azure Functions.
+* Identify and address memory-related performance issues.
+* Analyze trends in memory usage to optimize function performance.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Memory.png')} alt="Azure Functions Memory dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Network
+
+The **Azure Functions - Network** dashboard provides details on network traffic and connectivity related to your Azure Functions. This includes data on inbound and outbound traffic, network latency, and potential network issues.
+
+Use this dashboard to:
+* Track network activity and performance of your Azure Functions.
+* Diagnose network-related issues and connectivity problems.
+* Monitor network usage and optimize network configuration.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Network.png')} alt="Azure Functions Network dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Operations
+
+The **Azure Functions - Operations** dashboard provides an overview of operational metrics and activities related to your Azure Functions. This includes I/O Read, Write bytes/second and I/O Read, Write operations/second.
+
+Use this dashboard to:
+* Monitor the operational status and health of your Azure Functions.
+* Review deployment activities and execution metrics.
+* Analyze operational trends and optimize function management.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Operations.png')} alt="Azure Functions Operations dashboard" style={{border: '1px solid gray'}} width="800" />
```

Comment:
Crop the extra space in the dashboard image.

Reply:
Done